netsim/ansible/templates/vlan/cumulus.j2

```diff
@@ -18,11 +18,10 @@ auto bridge
 iface bridge
     bridge-vlan-aware yes
 {% if vlans is defined %}
-{%   for vdata in vlans.values() %}
-    bridge-vids {{ vdata.id }}
-{%   endfor %}
+{%  set vids = vlans.values() | map(attribute='id') | sort | map('string') %}
+    bridge-vids {{ ",".join(vids) }}
 {%  endif %}
-{% for ifdata in interfaces if ifdata.virtual_interface is not defined and ifdata.vlan is defined %}
+{% for ifdata in interfaces if ifdata.vlan is defined and ifdata.type!="svi" %}
```

Comment:
Did you run the integration tests? Are you sure this is safe?

Reply:
Struggle to run integration tests, the check is needed to include 'lag' type links but exclude 'svi' (like the virtual_interface check does)

Reply:
There might be other virtual interface types (we already have "tunnel" and "loopback"). Not sure it's a good idea to test if the interface type is not SVI.